### ISSUE TITLE
Remove the settings panel as a child and make it above the content

### ISSUE THREAD

**Issue by alobanov11** [artemnovichkov/figma-preview#2]
The current implementation narrows the content area, making it smaller than it actually is, so the layout is wildly divergent.

<img width="200" alt="Снимок экрана 2024-02-21 в 16 10 59" src="https://github.com/artemnovichkov/figma-preview/assets/5923243/dbfab3da-34d9-4e47-96ce-e89f31274664">

I would like to see the settings panel above content with opacity or something like this.

**Comment by artemnovichkov** [comment]
At first, thank you for using the library 🫰
I didn't add this feature but I thought about this case. It would be great to add it as a floating panel. I'm thinking about PiP panel behaviour in iOS, but for the first version floating and dragging will be enough. Do you want to make a pull request? Or it's just a feature request? 😅

**Comment by alobanov11** [comment]
Yeah, can you share your thoughts or design sketch how it should look so I can make changes and create PR?

**Comment by artemnovichkov** [comment]
Actually, I have no design. So you can use current implementation and make it floaty.
